### ISSUE TITLE
Create an action plan for Help articles logic

### ISSUE THREAD

**Issue by IndrekV** [toggl/toggldesktop#3330]
### 💻 Environment

Platform: All platforms

### 📒 Description
We have some code like `help_article_*`

Some time ago we had a plan to add more in-app help features to our apps. Making it easy to get all the needed info about desktop app from the feedback view. We did not finish this plan and we need to evaluate it and see if we want to finish it or remove the unused code.

The simple idea was that we fetch the contents of the desktop articles from the knowledgebase (it should be our support page + some known solutions to issues) and when user goes to feedback form and start to type we suggest links to articles with solutions or show a paragraph with text about the solution.

The aim of this is to give answers faster to our users and minimise the contacts from users that need a very simple answer.

References to old issues:
Save help articles into local database - #1620

**Comment by vedr4n** [comment]
First off, I like this initiative!

Considering support guides for desktop apps are just a single article for each platform (at the moment) I think it would be best to simply display that link in the Feedback form and then list 5-6 most frequently asked questions/answers. 

While this doesn't make for a perfect user experience, it will make article updates easier since they still need to happen only in 1 location. Also, most people don't read anyway. Just because the text can be there in the Desktop app doesn't mean people will read it. Dumb/easy questions happen only for 1 reason. We/people are lazy and we don't read. 

Another benefit of this approach is the ease of implementation. It will all be static text. Also, if they click on the KB article link (in the app) they'll still be able to easily reach out to Support. Every KB article has an embedded "Chat with Support" widget. 

If you're okay with this idea let me know and I'll ask Support to draft a list of about 5 most commonly asked questions by Desktop users. We can even make them platform-specific.

**Comment by IndrekV** [comment]
Yes, I like this idea. Go ahead and get the list of most common issues please.

Ideally, I would like the app to be also a bit intelligent in helping the user. We could track the error message user has in their log file and match these to common solutions and show the solutions in the feedback view if they open it.

This is sounding even cooler now.
